scripts/west_commands/sbom/data/spdx-licenses.yaml

```diff
@@ -364,6 +388,8 @@ FSFUL:
   name: "FSF Unlimited License"
 FSFULLR:
   name: "FSF Unlimited License (with License Retention)"
+FSFULLRWD:
+  name: "FSF Unlimited License (With License Retention    and Warranty Disclaimer)"
```

Comment:
```suggestion
  name: "FSF Unlimited License (With License Retention and Warranty Disclaimer)"
```

Reply:
> `(With License Retention    and Warranty Disclaimer)`

This is how it is in the official source SPDX database. I can look there if they have some easy was to report that, so maybe in future versions they will fix that. In the HTML report, multiple spaces are automatically merged into one, so this is not an issue for us.

Reply:
https://github.com/spdx/license-list-XML/issues/1908